tasks/internal/create-advisory-task/create-advisory-task.yaml

```diff
@@ -247,3 +286,53 @@ spec:
           # Construct the advisory url on customer portal to report back to the user as a result
           ADVISORY_TYPE=$(jq -r '.type' <<< "$ADVISORY_JSON" )
           ADVISORY_URL="${ADVISORY_URL_PREFIX}/${ADVISORY_TYPE}-${ADVISORY_NAME}"
+
+          # copy new advisory yaml so it can be saved as OCI artifact
+          cp -v "$ADVISORY_FILEPATH" /var/workdir
+    - name: skip-trusted-artifact-operations
+      ref:
```

Comment:
Are you sure stepActions are enabled in the app-sre cluster?

Reply:
Yes. Value from appsre staging:

`  enable-step-actions: 'true'`

Reply:
ok, thanks